### PR TITLE
Opersons in BDLaethb4

### PR DESCRIPTION
**Pull request by SusanneHummel**

### CODE REVIEW

File: new/PRS13193Giyorgis.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Giyorgis</persName>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS13193Giyorgis"
+                        passive="BDLaethb4">
+                        <desc>The name of Giyorgis is mentioned in the supplication formulas
+                            throughout the manuscript.</desc>
+                    </relation>
+                    <relation name="betmas:husbandOf" active="PRS13193Giyorgis"
+                        passive="PRS13195PRS13194AmataSell"/>
```
Comment (MassimoVilla) [inline new/PRS13193Giyorgis.xml:58]: ID to be fixed

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thanks!

File: new/PRS13193Giyorgis.xml (hunk 1 above)
Comment (SusanneHummel) [reply new/PRS13193Giyorgis.xml:58]: Thank you! too many numbers today ....

**Review by MassimoVilla — APPROVED** [review]

**Comment by thea-m** [comment]
(@SusanneHummel : There really is a second Gadla Giyorgis: https://betamasaheft.eu/persons/PRS13128GadlaGiyorgis/main)

**Pull request merged**